docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - /var/lib/postgresql
 
   hmpps-auth:
-    image: quay.io/hmpps/hmpps-auth:latest
+    image: quay.io/hmpps/hmpps-auth:2025-05-02.28494.105ac61
```

Comment:
Does this need putting into a separate PR so other PRs can still go through before this is all merged into main?

Reply:
will do 👍